osgar/drivers/kloubak.py

```diff
@@ -471,12 +472,15 @@ def process_packet(self, packet, verbose=False):
 #                    print(self.last_joint_angle, self.last_joint_angle2)
                     self.last_joint_angle, self.last_joint_angle2 = joint_rad(analog_joint_angle), joint_rad(analog_joint_angle2, second_ang=True)
                     self.publish('joint_angle', [round(math.degrees(self.last_joint_angle)*100), round(math.degrees(self.last_joint_angle2)*100)])
+                    self.publish('joint_angle_pose', [[[0, 0, 0], euler_to_quaternion(math.pi - self.last_joint_angle, 0, 0)],
+                                                      [[0, 0, 0], euler_to_quaternion(math.pi - self.last_joint_angle2, 0, 0)]])
```

Comment:
If you start driving K3 with this, it won't work. The ROS side expects a single pose in the message. The easiest way to make it work is to publish two separate messages to two channels here.